### DISCUSSION TITLE
How do i use this if my exam is tomorrow

### DISCUSSION THREAD

**Discussion by ShowcaseKing** [st3v3nmw/obsidian-spaced-repetition#420] [Q&A]
How to make my easy cards appear the least and the hard cards appear the most. I am looking for something similar to studysmarter.de.

**Comment by Ifeoluwapo49** [comment]
I'd suggest you use mochi.cards instead on desktop or mobile, at least until your exams are over. You can create flashcards directly from Obsidian (using "{{}}" as clozes and "---" as a separator for multiline flashcards. If you want to import multiple cards per .md file you can create a custom delimiter) and import them into Mochi for review. It allows you to review your cards in different ways. It even has a "Cram" function that allows you to learn all the cards in a deck in one go.

The best part is that, with a bit of tweaking, those .md files you made for Mochi will still work as flashcards with this plugin.
